webapp/data_structures/RELATIONALOP.js

```diff
@@ -26,86 +25,83 @@ function RELATIONALOP() {
+            this.x.model.sim=list(new ScilabString(["relationalop"]), new ScilabDouble([4]));
 		}
-                else if (this.Datatype==3|| this.Datatype==9) {
-                model.sim=list(new ScilabString(["relational_op_i32"]), new ScilabDouble([4]));
+        else if (this.Datatype==3|| this.Datatype==9) {
+            this.x.model.sim=list(new ScilabString(["relational_op_i32"]), new ScilabDouble([4]));
 		}
-                else if(this.Datatype==4) {
-                model.sim=list(new ScilabString(["relational_op_i16"]), new ScilabDouble([4]));
+        else if(this.Datatype==4) {
+            this.x.model.sim=list(new ScilabString(["relational_op_i16"]), new ScilabDouble([4]));
 		}
-                else if(this.Datatype==5) {
-                model.sim=list(new ScilabString(["relational_op_i8"]), new ScilabDouble([4]));
+        else if(this.Datatype==5) {
+            this.x.model.sim=list(new ScilabString(["relational_op_i8"]), new ScilabDouble([4]));
 		}
-                else if(this.Datatype==6) {
-                model.sim=list(new ScilabString(["relational_op_ui32"]), new ScilabDouble([4]));
+        else if(this.Datatype==6) {
+            this.x.model.sim=list(new ScilabString(["relational_op_ui32"]), new ScilabDouble([4]));
 		}
-                else if(this.Datatype==7) {
-                model.sim=list(new ScilabString(["relational_op_ui16"]), new ScilabDouble([4]));
+        else if(this.Datatype==7) {
+            this.x.model.sim=list(new ScilabString(["relational_op_ui16"]), new ScilabDouble([4]));
 		}
-                else if(this.Datatype==8){
-                model.sim=list(new ScilabString(["relational_op_ui8"]), new ScilabDouble([4]));
+        else if(this.Datatype==8){
+            this.x.model.sim=list(new ScilabString(["relational_op_ui8"]), new ScilabDouble([4]));
 		}
-                else {
-                alert("Datatype is not supported");
-		//RELATIONALOP.get();
-                }
-
-        var model = scicos_model();
-	model.sim = list(new ScilabString(["relationalop"]), new ScilabDouble([4]));
-        model.in = new ScilabDouble([1], [1]);
-        model.out = new ScilabDouble([1]);
-        model.ipar = new ScilabDouble([this.rule]);
-        model.blocktype = new ScilabString(["c"]);
-        model.dep_ut = new ScilabBoolean([true, false]);
-        var exprs = new ScilabString([this.rule],[this.zcr],[this.Datatype]);
-	this.displayParameter = [this.label];
-        var gr_i = new ScilabString(["xstringb(orig(1),orig(2),\"RELATIONALOP\",sz(1),sz(2));"]);
-        this.x = new standard_define(new ScilabDouble([2, 2]), model, exprs, gr_i);
+        else {
+            alert("Datatype is not supported");
+            throw "incorrect";
+		}
+        this.in = [[-1],[-1],[-2],[-2]];
+        this.out = [[-1],[-2]];
+        var io = set_io(this.x.model,this.x.graphics,this.in,this.out,[],[]);
+        this.x.model.ipar = new ScilabDouble([this.rule]);
```

Comment:
```suggestion
        model.ipar = new ScilabDouble([this.rule]);
```
use model here and below.